base/utils/config.go

```diff
@@ -169,7 +169,6 @@ func initConfig() {
 			Cfg.KafkaPayloadTrackerTopic = clowder.KafkaTopics[requestedKafkaPayloadTrackerTopic].Name
 		}
 	} else {
-		Cfg.KafkaBrokerAddress = GetEnv("KAFKA_BROKER_ADDRESS", "")
```

Comment:
We still want to support docker-compose functionality where providing the broker/list of brokers is done using env variable.